app/raid.js

```diff
@@ -324,7 +330,7 @@ class Raid {
 			}
 			else {
 				attendees_list += '<:PremierBall:347221891263496193>';
-			} //◻️, \t\t
+			}
```

Comment:
Hey! I wasn't sure if I wanted to get rid of those yet! :P that's fine, I can go back in history if I need it.  They were just there b/c I wasn't sure if people would accept the pokeball emoji's haha